roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -94,7 +89,18 @@
     - kubeadm_already_run.stat.exists
 
 - name: kubeadm | Initialize first master
-  command: timeout -k 600s 600s {{ bin_dir }}/kubeadm init --config={{ kube_config_dir }}/kubeadm-config.yaml --ignore-preflight-errors=all
+  command: >-
+    timeout -k 600s 600s
+    {{ bin_dir }}/kubeadm init
+    --config={{ kube_config_dir }}/kubeadm-config.yaml
+    --ignore-preflight-errors=all
+    {% if kubeadm_version is version('v1.14.0', '>=') %}
+    --experimental-upload-certs
+    {% endif %}
+    --skip-phases=addon/coredns
```

Comment:
I feel like this needs an additional `{% if kubeadm_version is version('v1.13.0...` check as `--skip-phases` was added in `1.13.0`:

https://github.com/kubernetes/kubernetes/pull/69684

If that's the case, should I open an issue?